discord/commands/context.py

```diff
@@ -136,51 +137,67 @@ async def invoke(
 
     @cached_property
     def channel(self) -> Optional[InteractionChannel]:
+        """Union[:class:`.abc.Messageable`]: Returns the channel associated with this context's command. Shorthand for :attr:`.Interaction.channel`.
+        """
         return self.interaction.channel
 
     @cached_property
     def channel_id(self) -> Optional[int]:
+        """`int`: Returns the ID of the channel associated with this context's command. Shorthand for :attr:`.Interaction.channel.id`."""
```

Comment:
pretty sure this should also have :class:
```suggestion
        """:class:`int`: Returns the ID of the channel associated with this context's command. Shorthand for :attr:`.Interaction.channel.id`."""
```